cvmfs/receiver/payload_processor.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_RECEIVER_PAYLOAD_PROCESSOR_H_
+#define CVMFS_RECEIVER_PAYLOAD_PROCESSOR_H_
+
+#include <stdint.h>
+#include <string>
+
+#include "pack.h"
+
+namespace receiver {
+
+class PayloadProcessor {
```

Comment:
I think it would be good to have a few words of documentation where the class is used.